src/test/groovy/org/osiam/test/integration/AbstractIT.groovy

```diff
@@ -62,8 +62,8 @@ abstract class AbstractIT extends Specification {
     protected AccessToken accessToken
 
     static {
-        OsiamConnector.setConnectTimeout(10000);
-        OsiamConnector.setReadTimeout(10000);
+        OsiamConnector.setConnectTimeout(Integer.parseInt(System.getProperty("debug.timeout.connect", "-1")));
+        OsiamConnector.setReadTimeout(Integer.parseInt(System.getProperty("debug.timeout.read", "-1")));
```

Comment:
Oh I get it. The timeouts have to be set to `-1`, because the tests will be started in debug mode within the IDE so the system properties set in the pom aren't available.